yarn-project/pxe/src/contract_function_simulator/execution_data_provider.ts

```diff
@@ -76,15 +75,15 @@ export interface ExecutionDataProvider {
    * Returns an object containing an array of note data.
    *
    * @param contractAddress - The contract address of the notes.
-   * @param owner - The owner of the notes.
+   * @param owner - The owner of the notes. If undefined, returns notes for all owners.
```

Comment:
Should we say: "If undefined, returns notes for all owners known to this PXE"?

"All owners" makes it feel like it's public